tests/Utils/Venus.js

```diff
@@ -181,7 +181,7 @@ async function makeVToken(opts = {}) {
       vToken = await saddle.getContractAt("VXvsLikeDelegate", vDelegator._address);
       break;
 
-    case "vbep20":
+    case 'vbep20':
```

Comment:
nit: At some point it would be nice to write this in typescript

Reply:
This does not include in this scope.

Reply:
yes this is why I have marked the comment as a nit

Reply:
I think this is also an unintended change. It should be double quotes